project/src/main/java/edu/depaul/cdm/se452/project/model/StudentRepository.java

```diff
@@ -7,9 +7,9 @@
 //repository class to help perform CRUD operations on the students table
 public interface StudentRepository extends CrudRepository<Student, Long> {
 	//find a student by passing an id
-	List<Student> findByStudentId(int num);
+	//List<Student> findByStudentId(int num);

```

Comment:
Why is this commented out?

Reply:
This wasn't working because i changed the naming and it was easier to comment it out just to get it to compile